<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>worldsettings</class>
 <widget class="QWidget" name="worldsettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>worldsettings</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>537</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>798</width>
      <height>2122</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="world">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>741</width>
       <height>521</height>
      </rect>
     </property>
     <property name="title">
      <string>World</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>19</y>
        <width>721</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="resources">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>741</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Resources</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>721</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="ResourcesGridLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="food">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>930</y>
       <width>741</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Food</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>721</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="FoodGridLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="animals">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1190</y>
       <width>741</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>Animals</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>721</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="AnimalsGridLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="monsters">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1680</y>
       <width>741</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>Monsters</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>721</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="MonstersGridLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
